.github/dockerfiles/ov_build/fedora_29/Dockerfile

```diff
@@ -110,3 +110,25 @@ RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | b
 RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
 ENV PATH="$NVM_DIR/versions/node/v${NODE_VERSION}/bin/:${PATH}"
 
+# To build git and curl from source
```

Comment:
These packages should not be installed in the first `dnf install` because if they are, they interfere with the building of Pythons.